common/modules/utils/src/lib.rs

```diff
@@ -67,7 +67,28 @@ pub trait UtilsModule {
         attr.into_part(&payment.amount)
     }
 
-    fn merge_from_payments<T: FixedSupplyToken<Self::Api> + Mergeable<Self::Api> + TopDecode>(
+    fn merge_from_payments_and_burn<
```

Comment:
it this used ?

Reply:
Yes, it is used in the merge_farm_tokens function. As it is a more common code, it was moved as a generic function in the utils modules, to make the contract more readable.